scripts/analysis/README.md

```diff
@@ -0,0 +1,93 @@
+# Source Analysis
```

Comment:
should we move these files into `script/get-usage` for consistency too?

Reply:
Good point. That's done.